staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -729,6 +732,13 @@ type IngressRuleValue struct {
 	// 2. Consider adding fields for ingress-type specific global options
 	// usable by a loadbalancer, like http keep-alive.
 
+	// HTTP is a pointer pointing to a list of http selectors.
```

Comment:
this is API-facing doc, so use the API field name (`http`) and don't mention pointer, which isn't meaningful in the REST API

Reply:
Thanks. I have modified them.